.github/workflows/main.yml

```
@@ -260,3 +266,14 @@ jobs:
         working-directory: test
         run: |
           ./run-system-test.sh
+          echo FINISHED SYSTEM TEST SCRIPT
+
+      - name: Dump ehr-api docker logs for debugging
+        if: ${{ always() }}
+        run: |
+          docker logs -t system-test-ehr-api-1 2>&1
+
+      - name: Dump imaging-api docker logs for debugging
+        if: ${{ always() }}
```

These are now failing because its running when the containers have been taken down
```suggestion
      - name: Dump ehr-api docker logs for debugging
        if: ${{ failure() }}
        run: |
          docker logs -t system-test-ehr-api-1 2>&1

      - name: Dump imaging-api docker logs for debugging
        if: ${{ failure() }}
```